### PR TITLE
add KILL statement can kill DDL jobs describe

### LINKED ISSUE #11069: KILL statement can kill DDL jobs since TiDB 6.2
## Change Request

1. Describe what you find is inappropriate or missing in the existing docs.
Starting from TiDB 6.2, `KILL` statement can kill DDL jobs. While TiDB 6.1 or older version `ADMIN CANCEL DDL` is required.
As far as I know, there are no document available which describes this change.

https://docs.pingcap.com/tidb/dev/sql-statement-admin-cancel-ddl
https://docs.pingcap.com/tidb/dev/sql-statement-kill

2. Describe your suggestion or addition.
Add some documents that `KILL` statement can kill DDL jobs since TiDB 6.2.

3. Provide some reference materials (documents, websites, etc) if you could.
https://github.com/pingcap/tidb/pull/35803 implemented that `KILL` statement can kill DDL jobs.

### PR DESCRIPTION
**Pull request by hey-hoho**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

fix #11069 
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-kill.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ aliases: ['/docs/dev/sql-statements/sql-statement-kill/','/docs/dev/reference/sq
 
 The `KILL` statement is used to terminate a connection in any TiDB instance in the current TiDB cluster.
 
+> **Note:**
+>
+> Starting from TiDB v6.2.0, `KILL` statement can kill DDL jobs.
```
Comment (dveeden) [inline sql-statements/sql-statement-kill.md:13]:
```suggestion
> Starting from TiDB v6.2.0, `KILL` statement is able to kill DDL jobs. 
```

File: sql-statements/sql-statement-admin-cancel-ddl.md [hunk 2]
```diff
@@ -37,6 +37,7 @@ If the operation fails to cancel the jobs, specific reasons are displayed.
 > - Only this operation can cancel DDL jobs. All other operations and environment changes (such as machine restart and cluster restart) cannot cancel these jobs.
 > - This operation can cancel multiple DDL jobs at the same time. You can get the ID of DDL jobs using the [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md) statement.
 > - If the jobs you want to cancel are finished, the cancellation operation fails.
+> - Starting from v6.2.0, [`KILL`](/sql-statements/sql-statement-kill.md) statement can kill DDL jobs.
```
Comment (dveeden) [inline sql-statements/sql-statement-admin-cancel-ddl.md:40]:
```suggestion
> - Starting from v6.2.0, [`KILL`](/sql-statements/sql-statement-kill.md) statement is able to kill DDL jobs. 
```

**Review by dveeden — APPROVED** [review]

**Review by dveeden — APPROVED** [review]

File: sql-statements/sql-statement-admin-cancel-ddl.md [hunk 3]
```diff
@@ -37,6 +37,7 @@ If the operation fails to cancel the jobs, specific reasons are displayed.
 > - Only this operation can cancel DDL jobs. All other operations and environment changes (such as machine restart and cluster restart) cannot cancel these jobs.
 > - This operation can cancel multiple DDL jobs at the same time. You can get the ID of DDL jobs using the [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md) statement.
 > - If the jobs you want to cancel are finished, the cancellation operation fails.
+> - Starting from v6.2.0, [`KILL`](/sql-statements/sql-statement-kill.md) statement is able to kill DDL jobs.
```
Comment (qiancai) [inline sql-statements/sql-statement-admin-cancel-ddl.md:40]:
```suggestion
> - Before v6.2.0, only this operation can cancel DDL jobs, and all other operations and environment changes (such as machine restart and cluster restart) cannot cancel these jobs. Starting from v6.2.0, [`KILL`](/sql-statements/sql-statement-kill.md) statement can also be used to cancel DDL jobs by killing them. 
> - This operation can cancel multiple DDL jobs at the same time. You can get the ID of DDL jobs using the [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md) statement.
> - If the jobs you want to cancel are finished, the cancellation operation fails.
```

File: sql-statements/sql-statement-kill.md [hunk 4]
```diff
@@ -8,6 +8,10 @@ aliases: ['/docs/dev/sql-statements/sql-statement-kill/','/docs/dev/reference/sq
 
 The `KILL` statement is used to terminate a connection in any TiDB instance in the current TiDB cluster.
 
+> **Note:**
+>
+> Starting from TiDB v6.2.0, `KILL` statement is able to kill DDL jobs.
```
Comment (qiancai) [inline sql-statements/sql-statement-kill.md:13]:
```suggestion
The `KILL` statement is used to terminate a connection in any TiDB instance in the current TiDB cluster. Starting from TiDB v6.2.0, you can also the `KILL` statement to kill DDL jobs. 
```

**Comment by qiancai** [comment]
Hi @hey-hoho, thanks for your contribution. I've added a few comments. PTAL. Thanks.

File: sql-statements/sql-statement-admin-cancel-ddl.md [hunk 5]
```diff
@@ -34,7 +34,7 @@ If the operation fails to cancel the jobs, specific reasons are displayed.
 
 > **Note:**
 >
-> - Only this operation can cancel DDL jobs. All other operations and environment changes (such as machine restart and cluster restart) cannot cancel these jobs.
+> - Before v6.2.0, only this operation can cancel DDL jobs, and all other operations and environment changes (such as machine restart and cluster restart) cannot cancel these jobs. Starting from v6.2.0, [`KILL`](/sql-statements/sql-statement-kill.md) statement can also be used to cancel DDL jobs by killing them.
```
Comment (qiancai) [inline sql-statements/sql-statement-admin-cancel-ddl.md:37]:
```suggestion
> - Before v6.2.0, only this operation can cancel DDL jobs, and all other operations and environment changes (such as machine restart and cluster restart) cannot cancel these jobs. Starting from v6.2.0, [`KILL`](/sql-statements/sql-statement-kill.md) statement can also be used to cancel ongoing DDL jobs by killing them. 
```

File: sql-statements/sql-statement-kill.md [hunk 6]
```diff
@@ -6,7 +6,7 @@ aliases: ['/docs/dev/sql-statements/sql-statement-kill/','/docs/dev/reference/sq
 
 # KILL
 
-The `KILL` statement is used to terminate a connection in any TiDB instance in the current TiDB cluster.
+The `KILL` statement is used to terminate a connection in any TiDB instance in the current TiDB cluster. Starting from TiDB v6.2.0, you can also the `KILL` statement to terminate DDL jobs.
```
Comment (qiancai) [inline sql-statements/sql-statement-kill.md:9]:
```suggestion
The `KILL` statement is used to terminate a connection in any TiDB instance in the current TiDB cluster. Starting from TiDB v6.2.0, you can also the `KILL` statement to terminate ongoing DDL jobs. 
```

File: sql-statements/sql-statement-kill.md [hunk 7]
```diff
@@ -6,7 +6,7 @@ aliases: ['/docs/dev/sql-statements/sql-statement-kill/','/docs/dev/reference/sq
 
 # KILL
 
-The `KILL` statement is used to terminate a connection in any TiDB instance in the current TiDB cluster.
+The `KILL` statement is used to terminate a connection in any TiDB instance in the current TiDB cluster. Starting from TiDB v6.2.0, you can also the `KILL` statement to terminate ongoing DDL jobs.
```
Comment (qiancai) [inline sql-statements/sql-statement-kill.md:9]:
```suggestion
The `KILL` statement is used to terminate a connection in any TiDB instance in the current TiDB cluster. Starting from TiDB v6.2.0, you can also use the `KILL` statement to terminate ongoing DDL jobs. 
```

**Review by dveeden — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

File: sql-statements/sql-statement-admin-cancel-ddl.md [hunk 8]
```diff
@@ -34,7 +34,7 @@ If the operation fails to cancel the jobs, specific reasons are displayed.
 
 > **Note:**
 >
-> - Only this operation can cancel DDL jobs. All other operations and environment changes (such as machine restart and cluster restart) cannot cancel these jobs.
+> - Before v6.2.0, only this operation can cancel DDL jobs, and all other operations and environment changes (such as machine restart and cluster restart) cannot cancel these jobs. Starting from v6.2.0, [`KILL`](/sql-statements/sql-statement-kill.md) statement can also be used to cancel ongoing DDL jobs by killing them.
```
Comment (ran-huang) [inline sql-statements/sql-statement-admin-cancel-ddl.md:37]:
```suggestion
> - Before v6.2.0, only this operation can cancel DDL jobs, and all other operations and environment changes (such as machine restart and cluster restart) cannot cancel these jobs. Starting from v6.2.0, [`KILL`](/sql-statements/sql-statement-kill.md) statements can also be used to cancel ongoing DDL jobs by killing them. 
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**